Casks/m/mega.rb

```diff
@@ -9,8 +9,19 @@
   homepage "https://megasoftware.net/"
 
   livecheck do
-    url "https://megasoftware.net/history"
-    regex(/MEGA\s\d+\sversion\s(\d+(?:\.\d+)+)/i)
+    url "https://www.megasoftware.net/current_release/"
+    strategy :json do |json|
+      json.map do |item|
+        next if item["operating_system"] != "mac" || item["release_type"] != "stable"
```

Comment:
```suggestion
        next if item["operating_system"] != "mac"
        next if item["release_type"] != "stable"
        next unless item["interface"].match?(/Graphical|GUI/i)
```

The JSON also includes versions for the command line app, so we should also restrict matching to the "Graphical (GUI)" entries. The versions seem to align for GUI and CLI but better safe than sorry.